src/App.js

```diff
@@ -1,15 +1,101 @@
 import React from 'react';
 import './App.css';
+import './components/TodoItem/TodoItem';
 
-import todos from './api/todos';
-import users from './api/users';
+import { todos } from './api/todos';
+import { users } from './api/users';
+import TodoList from './components/TodoList/TodoList';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Dynamic list of todos</h1>
-    </div>
-  );
+function getTodosWithUsers(todos, users) {
+  return todos.map(todo => ({
+    ...todo,
+    user: users.find(item => item.id === todo.userId),
+  }));
+}
+
+class App extends React.Component {
+  state = {
+    isLoading: false,
+    todos: [],
+    originTodos: [],
+    users: [],
+    isLoaded: false,
+  };
+
+  getData = () => {
+    this.setState({
+      isLoading: true,
+    });
+    todos().then(data => this.setState({
```

Comment:
use `Promise.all`

Reply:
on the 37th line of this file, I'm using Promise.all